database/src/main/resources/db/migration/V66__revurdering_add_attestering.sql

```diff
@@ -0,0 +1,7 @@
+alter table revurdering add column if not exists attestering jsonb default null;
+
+update revurdering
+set attestering = json_build_object('type', 'Iverksatt', 'attestant', attestant)
```

Comment:
Hvordan ser databaseobjektet ut når man persisterer null verdier? Vil de da være missing som her?

Reply:
Vet ikke om jag skjønte helt, men hvis en revurdering ikke har blitt iverksatt/underkjent så er `attestant`-feltet null i basen. Då blir `attestering`s-tabellen fortsatt null. 

Annars så er `attestant`-tabellen fyllt med en navIdent och den settes då på `attestant`-feltet i `Attestering`sobjektet. 😊 If that made sense? 😅

Reply:
Jeg trodde den Attestering.Iverksatt typen hadde fler felter enn attestant og type :shrug: men det ser det ut til at den ikke har :) Så nevermind.